transcription/transcription.go

```diff
@@ -38,6 +45,49 @@ func msgHeaders(from string, to []string, subject string) string {
 	return strings.Join(msgHeaders, "\r\n")
 }
 
+// StartTranscription transcribes a given file using Sphinx.
+// File name should be in "name.wav" format.
+func StartTranscription(fileName string, command string) (string, error) {
+
+	cmd := exec.Command("java", "-jar", command, fileName)
```

Comment:
Shouldn't you run `./gradlew`? Definitely try out your code to make sure it works (e.g. add a call to your function in `main.main`.